dali/operators/image/color/color_space_conversion.cu

```diff
@@ -211,6 +215,36 @@ void ColorSpaceConversion<GPUBackend>::RunImpl(DeviceWorkspace &ws) {
         // YCbCr -> GRAY
         detail::ConvertYCbCrToGray8uKernel<<<grid, block, 0, stream>>>(
           input_data, output_data, total_size);
+      } else if (conversion == kRGB_TO_Lab) {
+        // First from RGB to BGR
+        detail::ConvertRGBToBGR8uKernel<<<grid, block, 0, stream>>>(
+            input_data, output_data, total_size);
+        // Then from BGR to Lab
+        DALI_CHECK_NPP(
+              nppiBGRToLab_8u_C3R(
+                output_data, nStepOutput, output_data, nStepOutput, size));
+      } else if (conversion == kBGR_TO_Lab) {
+        // BGR -> Lab
+        DALI_CHECK_NPP(
+              nppiBGRToLab_8u_C3R(
+                output_data, nStepOutput, output_data, nStepOutput, size));
+      } else if (conversion == kLab_TO_RGB || conversion == kLab_TO_BGR) {
```

Comment:
This condition will catch `kLab_TO_RGB` and we'll never convert from BGR to RGB as we should.

Reply:
this time the PR is a little careless. i would modify with build, test try my best.

Reply:
fixed.